src/bosh-director/lib/bosh/director/models/vm.rb

```diff
@@ -19,6 +19,16 @@ def network_spec=(spec)
     end
 
     def ips
+      ips_cidr.map do | cidr_ip |
+       if ( cidr_ip.include?(':') && cidr_ip.include?('/128') ) || ( cidr_ip.include?('.')  && cidr_ip.include?('/32') )
```

Comment:
Can `Network::IPV6_DEFAULT_PREFIX_SIZE` and `Network::IPV4_DEFAULT_PREFIX_SIZE` be used here to avoid magic numbers?